load_config.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
To make it easy to load MessageHub and Elasticsearch credentials and run locally and in docker